src/components/Navigation.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from '../styles/Navigation.module.css';
+
+export default function Navigation() {
+  return (
+    <header>
+      <div className={styles['nav-wrapper']}>
+        <h2>Bookstore CMS</h2>
+        <nav>
+          <ul className={styles['nav-links']}>
+            <li>
+              <NavLink to="/">BOOKS</NavLink>
+            </li>
+            <li>
+              <NavLink to="/categories">CATEGORIES</NavLink>
+            </li>
```

Comment:
- [ ] Great job adding the styling to the bookstore. Kindly note that your header is missing the profile icon. Since you are expected to stick to design as much as possible, kindly add this, please.
![image](https://github.com/Enoisong/book_store/assets/94551484/0a797ebe-ed30-470e-887c-f78a280a4679)